MODULE.bazel

```diff
@@ -835,3 +835,12 @@ http_file(
     sha256 = "ffe807170f06b579098f948eca42ba1ea1fd2cb7b8290c9d41101efe976337f7",
     url = "https://github.com/nmattia/rustix/commit/330bcc07796f4b8e6443987fbb1b9be79c446bd2.diff",
 )
+
+# Management canister candid interface
+
+http_file(
+    name = "management_canister_did",
+    downloaded_file_path = "ic.did",
+    sha256 = "31d4654d60b364420a2e52f546f06b2255dc78ac8c2d768271f004b8946e92cb",
+    url = "https://raw.githubusercontent.com/dfinity/interface-spec/b7cbca243f527b6819a0b0f644f87366a47d3eb4/spec/_attachments/ic.did",
+)
```

Comment:
Do we need both this and [`interface_spec` above](https://sourcegraph.com/github.com/dfinity/ic/-/blob/MODULE.bazel?L793-L799)? If so, could you place them side by side to easily spot the differences?

Reply:
Thank you for pointing this out! I wasn't aware of the other dependency used in fuzzers so I moved them both together. I'd prefer to not bump it in this PR if you don't mind.

Reply:
@venkkatesh-sekar Should we keep two versions of this external dependency or unify them to the latest master commit in this PR after all?

Reply:
@mraszyk the fuzzers can use the latest master version as well as they are only used for corpus generation.

Reply:
Great! I've merged them together into a single dependency pulling the latest version.